pkg/build/generator/generator.go

```diff
@@ -235,6 +236,11 @@ func (g *BuildGenerator) Instantiate(ctx kapi.Context, request *buildapi.BuildRe
 		return nil, err
 	}
 
+	// add labels and annotations from the buildrequest.  existing label/annotations will take
```

Comment:
changed.

On Tue, Apr 12, 2016 at 3:49 AM, Michal Fojtik <notifications@github.com>
wrote:

> In pkg/build/generator/generator.go
> <https://github.com/openshift/origin/pull/8405#discussion_r59333822>:
>
> > @@ -235,6 +236,11 @@ func (g *BuildGenerator) Instantiate(ctx kapi.Context, request *buildapi.BuildRe
> >  		return nil, err
> >  	}
> >
> > +	// add labels and annotations from the buildrequest.  existing label/annotations will take
>
> nit: capitalize 'Add' ;-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8405/files/92ecffb3fe73de3f18a98486624aacf1c313df1b#r59333822>
>



-- 
Ben Parees | OpenShift